mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -278,7 +316,11 @@ void AbstractControllerExecution::run()
   last_valid_cmd_time_ = ros::Time();
   int retries = 0;
   int seq = 0;
+  first_ignored_time_ = ros::Time::now();
 
+  // check if the velocity ignore check is enabled or not
+  bool robot_ignore_vel_enabled = (robot_ignore_check_tolerance_ > 0.0);
```

Comment:
```suggestion
```
S1: you don't really need this variable here, just check directly inside the if statement below for `robot_ignore_check_tolerance_ > 0.0`

S2: move this logic inside `checkVelocityIgnore`. So right at the top of the function you can check if the tolerance is set to > 0 and if not just immediately return false